### PR TITLE
Update o365-data-locations.md

### PR DESCRIPTION
**Pull request by dmwmsft**
added covid 19 statement.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/o365-data-locations.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ Find information about the contractual commitments for the storage location of c
 
 For Azure Active Directory data locations, please visit [AAD Data Map](https://go.microsoft.com/fwlink/p/?linkid=2092972).
 
+
+Due to the unprecedented circumstances around COVID-19 and the need to manage online services demand in Europe, if your organization is an educational institution or has selected a free Teams offering, we may provision your Office 365 tenant in the European Union (EU), European Free Trade Association (EFTA), the United Kingdom (UK), United States (US), or Canada (CA), or transfer your data to any data centers in the EU, EFTA, UK, US or CA.  This means that your data may be stored at rest within the EU, EFTA, UK, US or CA.  Our commitment to compliance, privacy, and security remains unchanged while we implement necessary and prudent measures to maintain service availability.  We will comply with all applicable law with respect to data transfers.
```
Comment (JohanFreelancer9) [inline Enterprise/o365-data-locations.md]:
```suggestion
Due to the unprecedented circumstances around COVID-19 and the need to manage online services demand in Europe, if your organization is an educational institution or has selected a free Teams offering, we may provision your Office 365 tenant in the European Union (EU), European Free Trade Association (EFTA), the United Kingdom (UK), United States (US), or Canada (CA), or transfer your data to any data centers in the EU, EFTA, UK, US, or CA. This means that your data may be stored at rest within the EU, EFTA, UK, US, or CA. Our commitment to compliance, privacy, and security remains unchanged while we implement necessary and prudent measures to maintain service availability.  We will comply with all applicable laws with respect to data transfers.
```

**Comment by dmwmsft** [comment]
Do I need to do something to approve his changes?  I do not see it live yet.

**Comment by mypil** [comment]
@dmwmsft - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by dmwmsft** [comment]
> @dmwmsft - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

let me know if I did it correctly.  I did not see a "merge" option.

**Pull request merged**